cli/src/main/java/dev/starfix/Starfix.java

```diff
@@ -124,10 +124,11 @@ public int cloneCmd(@Parameters(index = "0") String url) {
     @Override
     public void run() {
         if(uri==null||uri.isEmpty())
-            {
-                System.out.println("Empty args..");
-                return;
-            }
+        {   
+            System.out.println("\n === No/Empty Arguments received === ");
```

Comment:
we can just remove that - normal to just show usage.

Reply:
Removed :+1: